SingularityUI/app/views/taskHealthcheckNotificationSubview.coffee

```diff
@@ -18,12 +18,24 @@ class taskHealthcheckNotificationSubview extends View
         @listenTo @model, 'sync', @render
         @listenTo @pendingDeploys, 'sync', @render
 
+    deployFailureKilledTask: =>
```

Comment:
Doesn't look like this method is used. You can probably eliminate the section below where it is calculated in `renderData` and just use this